### ISSUE TITLE
Why Doesn't Yalp Show All The Same Search Results As Play Store?

### ISSUE THREAD

**Issue by 0pLuS0** [yeriomin/YalpStore#587]
I thought that Yalp connected to the Play Store server and would show all the same search results and it doesn't, I noticed Aurora Store does the same thing.

If neither of these apps can show all the apps on the Play Store then these applications are not helping very much...

I did a search for ```Calendars``` as and example in Yalp and it only gave 9 results back, have a look at the Search results on the Play Store;

https://play.google.com/store/search?q=Calendar&c=apps&authuser

Can someone please tell me what is going on?

All the time I've been using Aurora Store or Yalp I thought these would be better alternatives to the Play Store, but this is not turning out to be true when you can't even see all the applications to download....

This is the same no matter what you search for; 9-10 video players, email clients, etc...

This is really bad, we need this fixed...

I hope the DEV here will reply to this, I want to know why this happening and what the solution is for this?

THANKS

**Comment by 0pLuS0** [comment]
I went back checking the ```Search Filters``` everything is on Default settings, and this time I did a search on the word ```Gallery```, again the same issue, Yalp only shows 9 results.

If I got to Play Store and do a search on the word ```Gallery``` there around 70 results...

What is going on here?

I personally don't want to use the Play Store, but Aurora & Yalp both seem to have this same problem, and if this isn't fixed, then these apps are not helping very much to show us all the available apps.

What's the point of having Aurora Or Yalp when they always show only 5%-10% of the actual results...

hmm

**Comment by 0pLuS0** [comment]
@yeriomin Hi

I went back checking the ```Search Filters``` everything is on Default settings, and this time I did a search on the word ```Gallery```, again the same issue, Yalp only shows 9 results.

If I go to Play Store and do a search on the word ```Gallery``` there around 70 results...

What is going on here?

I personally don't want to use the Play Store, but Aurora & Yalp both seem to have this same problem, and if this isn't fixed, then these apps are not helping very much to show us all the available apps to download.

What's the point of having Aurora Or Yalp when they always show only 5%-10% of the actual results...

hmm
